hudi-common/src/main/java/org/apache/hudi/common/table/log/block/HoodieHFileDataBlock.java

```diff
@@ -74,11 +61,10 @@
  * base file format.
  */
 public class HoodieHFileDataBlock extends HoodieDataBlock {
+  public static final String HFILE_COMPRESSION_ALGO_PARAM_KEY = "hfile_compression_algo";
```

Comment:
Why don't we have a hoodie config for this like we have with parquet? I would rather just make the compression enum one of the params to serializeRecordsToLogBlock rather than it being inside the map

Reply:
Fixed by using `HFILE_COMPRESSION_ALGORITHM_NAME.key()` directly.  Also, I directly pass the String value of the config down so the String value is directly converted to the corresponding `Compression.Algorithm`, like `ParquetUtils`.